docs/src/fpga.md

```diff
@@ -8,9 +8,12 @@ The machine that is used for development and cross compilation should have:
 
 - Rust
 - Docker
+  - Pull the FPGA build image: `$ docker pull ghcr.io/chipsalliance/caliptra-build-image:latest` 
 - rsync
 - git
 
+**NOTE:** Setup rootless docker. Here are the steps [here](https://docs.docker.com/engine/security/rootless/). 
```

Comment:
Alternatively, in the future we could consider using podman. (Just noting here; not a blocker for this PR).